Fixing Links on the InnerSource in Action Page

clcoffey opened the pull request:
This relates to issue: https://github.com/InnerSourceCommons/InnerSourceMarketing/issues/149

There are still 4 that need to be addressed. @Katiegilroyy if you could have a look.

rrrutledge reviewed (approved):
Thanks, Georg!

lenucksi commented:
Travis failures are due to required Ruby platform updates as it looks like -> 2.4.9 to 2.5.0.

rrrutledge reviewed (approved):
Great, Georg!

dicortazar commented:
Checks seem to fail. @lenucksi any help here? :)

lenucksi commented:
> 
> 
> Checks seem to fail. @lenucksi any help here? :)

Two things: 
- Travis announced deactivation of its OSS plan, i.e. Travis is dead for our use.
- Also, as the logs say: The dependencies need a higher Ruby version. The current setup doesn't have it. Solution is to migrate to to GH Actions, see https://github.com/InnerSourceCommons/innersourcecommons.org/pull/233

rrrutledge commented:
I'm thinking of merging just to get the notes in _master_ since these failures don't seem related to the proposed change.  Sound reasonable?

lenucksi commented:
> 
> 
> I'm thinking of merging just to get the notes in _master_ since these failures don't seem related to the proposed change. Sound reasonable?

Totally. They will not build onto the website anymore though since the Travis process is broken. Someone will have to take flip things over to the GHActions rendered version in this repos settings using the other branch as website source.

clcoffey committed:
Fixing Links on the InnerSource in Action Page

This relates to issue: https://github.com/InnerSourceCommons/InnerSourceMarketing/issues/149

There are still 4 that need to be addressed.

claredillon reviewed (approved):
Thanks Cristina! Looks good.
